genesyscloud/resource_exporter/resource_name_sanitizer.go

```diff
@@ -18,24 +20,36 @@ type Sanitizer interface {
 
 // Two different Sanitizer structs one with the original algorithmn
 type sanitizerOriginal struct{}
-type sanitizerOptimized struct{}
+type sanitizerNameOptimized struct{}
+type sanitizerTimeOptimized struct{}
 
 // NewSanitizierProvider returns a Sanitizer. Without a GENESYS_SANITIZER_LEGACY environment variable set it will always use the optimized Sanitizer
 func NewSanitizerProvider() *SanitizerProvider {
 	// Check if the environment variable is set
-	_, exists := os.LookupEnv("GENESYS_SANITIZER_LEGACY")
```

Comment:
We should be putting feature toggle OS lookups directly in the code.  Take a look at util/FeatureToggles pkg for how to setup a feature toggle.

Reply:
Fixed